[cloudera@quickstart ~]$ ^C
[cloudera@quickstart ~]$ sqoop import \
> > --connect jdbc:mysql://quickstart.cloudera/retail_db \
> > --username root --password cloudera \
> > --table departments \
> > --field-terminated-by '\t'
18/01/21 13:34:15 INFO sqoop.Sqoop: Running Sqoop version: 1.4.6-cdh5.12.0
18/01/21 13:34:15 ERROR tool.BaseSqoopTool: Error parsing arguments for import:
18/01/21 13:34:15 ERROR tool.BaseSqoopTool: Unrecognized argument: jdbc:mysql://quickstart.cloudera/retail_db
18/01/21 13:34:15 ERROR tool.BaseSqoopTool: Unrecognized argument: root
18/01/21 13:34:15 ERROR tool.BaseSqoopTool: Unrecognized argument: --password
18/01/21 13:34:15 ERROR tool.BaseSqoopTool: Unrecognized argument: cloudera
18/01/21 13:34:15 ERROR tool.BaseSqoopTool: Unrecognized argument: departments
18/01/21 13:34:15 ERROR tool.BaseSqoopTool: Unrecognized argument: \t

Try --help for usage instructions.
[cloudera@quickstart ~]$ sqoop import \
> > --connect jdbc:mysql://quickstart.cloudera/retail_db \
> > --username root --password cloudera \
> > --table departments \
> > --field-terminated-by '\t'
18/01/21 13:34:46 INFO sqoop.Sqoop: Running Sqoop version: 1.4.6-cdh5.12.0
18/01/21 13:34:46 ERROR tool.BaseSqoopTool: Error parsing arguments for import:
18/01/21 13:34:46 ERROR tool.BaseSqoopTool: Unrecognized argument: jdbc:mysql://quickstart.cloudera/retail_db
18/01/21 13:34:46 ERROR tool.BaseSqoopTool: Unrecognized argument: root
18/01/21 13:34:46 ERROR tool.BaseSqoopTool: Unrecognized argument: --password
18/01/21 13:34:46 ERROR tool.BaseSqoopTool: Unrecognized argument: cloudera
18/01/21 13:34:46 ERROR tool.BaseSqoopTool: Unrecognized argument: departments
18/01/21 13:34:46 ERROR tool.BaseSqoopTool: Unrecognized argument: \t

Try --help for usage instructions.
[cloudera@quickstart ~]$ sqoop import \
> --connect jdbc:mysql://quickstart.cloudera/retail_db \
> --username root --password cloudera \
> --table departments \
> --field-terminated-by '\t'
Warning: /usr/lib/sqoop/../accumulo does not exist! Accumulo imports will fail.
Please set $ACCUMULO_HOME to the root of your Accumulo installation.
18/01/21 13:35:37 INFO sqoop.Sqoop: Running Sqoop version: 1.4.6-cdh5.12.0
18/01/21 13:35:37 WARN tool.BaseSqoopTool: Setting your password on the command-line is insecure. Consider using -P instead.
18/01/21 13:35:37 ERROR tool.BaseSqoopTool: Error parsing arguments for import:
18/01/21 13:35:37 ERROR tool.BaseSqoopTool: Unrecognized argument: --field-terminated-by
18/01/21 13:35:37 ERROR tool.BaseSqoopTool: Unrecognized argument: \t

Try --help for usage instructions.
[cloudera@quickstart ~]$ sqoop import \
> --connect jdbc:mysql://quickstart.cloudera/retail_db \
> --username root --password cloudera \
> --table departments \
> --field-terminated-by '\t'
Warning: /usr/lib/sqoop/../accumulo does not exist! Accumulo imports will fail.
Please set $ACCUMULO_HOME to the root of your Accumulo installation.
18/01/21 13:36:00 INFO sqoop.Sqoop: Running Sqoop version: 1.4.6-cdh5.12.0
18/01/21 13:36:00 WARN tool.BaseSqoopTool: Setting your password on the command-line is insecure. Consider using -P instead.
18/01/21 13:36:00 ERROR tool.BaseSqoopTool: Error parsing arguments for import:
18/01/21 13:36:00 ERROR tool.BaseSqoopTool: Unrecognized argument: --field-terminated-by
18/01/21 13:36:00 ERROR tool.BaseSqoopTool: Unrecognized argument: \t

Try --help for usage instructions.
[cloudera@quickstart ~]$ sqoop import\
> --connect jdbc:mysql://quickstart.cloudera/retail_db \
> --username root --password cloudera \
> --table departments \
> --fields-terminated-by '\t'
Warning: /usr/lib/sqoop/../accumulo does not exist! Accumulo imports will fail.
Please set $ACCUMULO_HOME to the root of your Accumulo installation.
No such sqoop tool: import--connect. See 'sqoop help'.
[cloudera@quickstart ~]$ sqoop import \
> --connect jdbc:mysql://quickstart.cloudera/retail_db \
> --username root --password cloudera \
> --table departments \
> --fields-terminated-by '\t'
Warning: /usr/lib/sqoop/../accumulo does not exist! Accumulo imports will fail.
Please set $ACCUMULO_HOME to the root of your Accumulo installation.
18/01/21 13:41:26 INFO sqoop.Sqoop: Running Sqoop version: 1.4.6-cdh5.12.0
18/01/21 13:41:26 WARN tool.BaseSqoopTool: Setting your password on the command-line is insecure. Consider using -P instead.
18/01/21 13:41:26 INFO manager.MySQLManager: Preparing to use a MySQL streaming resultset.
18/01/21 13:41:26 INFO tool.CodeGenTool: Beginning code generation
18/01/21 13:41:27 INFO manager.SqlManager: Executing SQL statement: SELECT t.* FROM `departments` AS t LIMIT 1
18/01/21 13:41:27 INFO manager.SqlManager: Executing SQL statement: SELECT t.* FROM `departments` AS t LIMIT 1
18/01/21 13:41:27 INFO orm.CompilationManager: HADOOP_MAPRED_HOME is /usr/lib/hadoop-mapreduce
Note: /tmp/sqoop-cloudera/compile/e555eb0d88378a284821092d4077b97b/departments.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
18/01/21 13:41:30 INFO orm.CompilationManager: Writing jar file: /tmp/sqoop-cloudera/compile/e555eb0d88378a284821092d4077b97b/departments.jar
18/01/21 13:41:30 WARN manager.MySQLManager: It looks like you are importing from mysql.
18/01/21 13:41:30 WARN manager.MySQLManager: This transfer can be faster! Use the --direct
18/01/21 13:41:30 WARN manager.MySQLManager: option to exercise a MySQL-specific fast path.
18/01/21 13:41:30 INFO manager.MySQLManager: Setting zero DATETIME behavior to convertToNull (mysql)
18/01/21 13:41:30 INFO mapreduce.ImportJobBase: Beginning import of departments
18/01/21 13:41:30 INFO Configuration.deprecation: mapred.job.tracker is deprecated. Instead, use mapreduce.jobtracker.address
18/01/21 13:41:31 INFO Configuration.deprecation: mapred.jar is deprecated. Instead, use mapreduce.job.jar
18/01/21 13:41:33 INFO Configuration.deprecation: mapred.map.tasks is deprecated. Instead, use mapreduce.job.maps
18/01/21 13:41:33 INFO client.RMProxy: Connecting to ResourceManager at /0.0.0.0:8032
18/01/21 13:41:35 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:41:35 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:41:35 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:41:35 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:41:37 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:41:37 INFO db.DBInputFormat: Using read commited transaction isolation
18/01/21 13:41:37 INFO db.DataDrivenDBInputFormat: BoundingValsQuery: SELECT MIN(`department_id`), MAX(`department_id`) FROM `departments`
18/01/21 13:41:37 INFO db.IntegerSplitter: Split size: 1; Num splits: 4 from: 2 to: 7
18/01/21 13:41:37 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:41:37 INFO mapreduce.JobSubmitter: number of splits:4
18/01/21 13:41:37 INFO mapreduce.JobSubmitter: Submitting tokens for job: job_1516563730194_0002
18/01/21 13:41:38 INFO impl.YarnClientImpl: Submitted application application_1516563730194_0002
18/01/21 13:41:38 INFO mapreduce.Job: The url to track the job: http://quickstart.cloudera:8088/proxy/application_1516563730194_0002/
18/01/21 13:41:38 INFO mapreduce.Job: Running job: job_1516563730194_0002
18/01/21 13:42:02 INFO mapreduce.Job: Job job_1516563730194_0002 running in uber mode : false
18/01/21 13:42:02 INFO mapreduce.Job:  map 0% reduce 0%
18/01/21 13:42:39 INFO mapreduce.Job:  map 50% reduce 0%
18/01/21 13:42:44 INFO mapreduce.Job:  map 100% reduce 0%
18/01/21 13:42:44 INFO mapreduce.Job: Job job_1516563730194_0002 completed successfully
18/01/21 13:42:44 INFO mapreduce.Job: Counters: 31
	File System Counters
		FILE: Number of bytes read=0
		FILE: Number of bytes written=605812
		FILE: Number of read operations=0
		FILE: Number of large read operations=0
		FILE: Number of write operations=0
		HDFS: Number of bytes read=481
		HDFS: Number of bytes written=60
		HDFS: Number of read operations=16
		HDFS: Number of large read operations=0
		HDFS: Number of write operations=8
	Job Counters 
		Killed map tasks=1
		Launched map tasks=4
		Other local map tasks=4
		Total time spent by all maps in occupied slots (ms)=137894
		Total time spent by all reduces in occupied slots (ms)=0
		Total time spent by all map tasks (ms)=137894
		Total vcore-milliseconds taken by all map tasks=137894
		Total megabyte-milliseconds taken by all map tasks=141203456
	Map-Reduce Framework
		Map input records=6
		Map output records=6
		Input split bytes=481
		Spilled Records=0
		Failed Shuffles=0
		Merged Map outputs=0
		GC time elapsed (ms)=1232
		CPU time spent (ms)=3900
		Physical memory (bytes) snapshot=463577088
		Virtual memory (bytes) snapshot=6042701824
		Total committed heap usage (bytes)=243007488
	File Input Format Counters 
		Bytes Read=0
	File Output Format Counters 
		Bytes Written=60
18/01/21 13:42:44 INFO mapreduce.ImportJobBase: Transferred 60 bytes in 71.6812 seconds (0.837 bytes/sec)
18/01/21 13:42:44 INFO mapreduce.ImportJobBase: Retrieved 6 records.
[cloudera@quickstart ~]$ sqoop import \
> connect jdbc:mysql://quickstart.cloudera/retail_db \
> --username root --password cloudera \
> --table products \
> --fields-terminated-by '\t' \
> -m1 \
> --target-dir /itemsdata
Warning: /usr/lib/sqoop/../accumulo does not exist! Accumulo imports will fail.
Please set $ACCUMULO_HOME to the root of your Accumulo installation.
18/01/21 13:48:23 INFO sqoop.Sqoop: Running Sqoop version: 1.4.6-cdh5.12.0
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Error parsing arguments for import:
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: connect
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: jdbc:mysql://quickstart.cloudera/retail_db
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: --username
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: root
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: --password
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: cloudera
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: --table
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: products
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: --fields-terminated-by
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: \t
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: -m
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: 1
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: --target-dir
18/01/21 13:48:23 ERROR tool.BaseSqoopTool: Unrecognized argument: /itemsdata

Try --help for usage instructions.
[cloudera@quickstart ~]$ sqoop import \
>  --connect jdbc:mysql://quickstart.cloudera/retail_db \
>  --username root --password cloudera \
>  --table products \
>  --fields-terminated-by '\t' \
>  -m1 \
>  --target-dir /itemsdata
Warning: /usr/lib/sqoop/../accumulo does not exist! Accumulo imports will fail.
Please set $ACCUMULO_HOME to the root of your Accumulo installation.
18/01/21 13:48:38 INFO sqoop.Sqoop: Running Sqoop version: 1.4.6-cdh5.12.0
18/01/21 13:48:38 WARN tool.BaseSqoopTool: Setting your password on the command-line is insecure. Consider using -P instead.
18/01/21 13:48:39 INFO manager.MySQLManager: Preparing to use a MySQL streaming resultset.
18/01/21 13:48:39 INFO tool.CodeGenTool: Beginning code generation
18/01/21 13:48:39 INFO manager.SqlManager: Executing SQL statement: SELECT t.* FROM `products` AS t LIMIT 1
18/01/21 13:48:40 INFO manager.SqlManager: Executing SQL statement: SELECT t.* FROM `products` AS t LIMIT 1
18/01/21 13:48:40 INFO orm.CompilationManager: HADOOP_MAPRED_HOME is /usr/lib/hadoop-mapreduce
Note: /tmp/sqoop-cloudera/compile/7c99d713d8d477d68d13196bb7098e4a/products.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
18/01/21 13:48:43 INFO orm.CompilationManager: Writing jar file: /tmp/sqoop-cloudera/compile/7c99d713d8d477d68d13196bb7098e4a/products.jar
18/01/21 13:48:43 WARN manager.MySQLManager: It looks like you are importing from mysql.
18/01/21 13:48:43 WARN manager.MySQLManager: This transfer can be faster! Use the --direct
18/01/21 13:48:43 WARN manager.MySQLManager: option to exercise a MySQL-specific fast path.
18/01/21 13:48:43 INFO manager.MySQLManager: Setting zero DATETIME behavior to convertToNull (mysql)
18/01/21 13:48:43 INFO mapreduce.ImportJobBase: Beginning import of products
18/01/21 13:48:43 INFO Configuration.deprecation: mapred.job.tracker is deprecated. Instead, use mapreduce.jobtracker.address
18/01/21 13:48:44 INFO Configuration.deprecation: mapred.jar is deprecated. Instead, use mapreduce.job.jar
18/01/21 13:48:45 INFO Configuration.deprecation: mapred.map.tasks is deprecated. Instead, use mapreduce.job.maps
18/01/21 13:48:45 INFO client.RMProxy: Connecting to ResourceManager at /0.0.0.0:8032
18/01/21 13:48:46 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:47 WARN hdfs.DFSClient: Caught exception 
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at java.lang.Thread.join(Thread.java:1281)
	at java.lang.Thread.join(Thread.java:1355)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.closeResponder(DFSOutputStream.java:952)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.endBlock(DFSOutputStream.java:690)
	at org.apache.hadoop.hdfs.DFSOutputStream$DataStreamer.run(DFSOutputStream.java:879)
18/01/21 13:48:48 INFO db.DBInputFormat: Using read commited transaction isolation
18/01/21 13:48:48 INFO mapreduce.JobSubmitter: number of splits:1
18/01/21 13:48:48 INFO mapreduce.JobSubmitter: Submitting tokens for job: job_1516563730194_0003
18/01/21 13:48:49 INFO impl.YarnClientImpl: Submitted application application_1516563730194_0003
18/01/21 13:48:49 INFO mapreduce.Job: The url to track the job: http://quickstart.cloudera:8088/proxy/application_1516563730194_0003/
18/01/21 13:48:49 INFO mapreduce.Job: Running job: job_1516563730194_0003
18/01/21 13:49:00 INFO mapreduce.Job: Job job_1516563730194_0003 running in uber mode : false
18/01/21 13:49:00 INFO mapreduce.Job:  map 0% reduce 0%
18/01/21 13:49:08 INFO mapreduce.Job:  map 100% reduce 0%
18/01/21 13:49:08 INFO mapreduce.Job: Job job_1516563730194_0003 completed successfully
18/01/21 13:49:08 INFO mapreduce.Job: Counters: 30
	File System Counters
		FILE: Number of bytes read=0
		FILE: Number of bytes written=151633
		FILE: Number of read operations=0
		FILE: Number of large read operations=0
		FILE: Number of write operations=0
		HDFS: Number of bytes read=87
		HDFS: Number of bytes written=173993
		HDFS: Number of read operations=4
		HDFS: Number of large read operations=0
		HDFS: Number of write operations=2
	Job Counters 
		Launched map tasks=1
		Other local map tasks=1
		Total time spent by all maps in occupied slots (ms)=5976
		Total time spent by all reduces in occupied slots (ms)=0
		Total time spent by all map tasks (ms)=5976
		Total vcore-milliseconds taken by all map tasks=5976
		Total megabyte-milliseconds taken by all map tasks=6119424
	Map-Reduce Framework
		Map input records=1345
		Map output records=1345
		Input split bytes=87
		Spilled Records=0
		Failed Shuffles=0
		Merged Map outputs=0
		GC time elapsed (ms)=96
		CPU time spent (ms)=1110
		Physical memory (bytes) snapshot=136597504
		Virtual memory (bytes) snapshot=1510182912
		Total committed heap usage (bytes)=60751872
	File Input Format Counters 
		Bytes Read=0
	File Output Format Counters 
		Bytes Written=173993
18/01/21 13:49:08 INFO mapreduce.ImportJobBase: Transferred 169.915 KB in 23.396 seconds (7.2626 KB/sec)
18/01/21 13:49:08 INFO mapreduce.ImportJobBase: Retrieved 1345 records.
